### PR TITLE
Fix/make version be flag in CLI && write errors instead of "help" behavior

### LINKED ISSUE #571: capricious neofs-cli keys
## Expected Behavior
1. I'm able to print neofs-cli version two ways: `neofs-cli version` and `neofs-cli --version`
2. I'm able to print help topic for netmap two ways: `neofs-cli netmap help` and `neofs-cli netmap --help`

## Current Behavior
1.
```
$ neofs-cli version
Version: v0.20.0 
Build: 2021-05-21T10:04:40+00:00 
Debug: false

$ neofs-cli --version
Error: unknown flag: --version
Usage:
  neofs-cli [command]
...
and long help message
```
2.
```
$ neofs-cli netmap help
netmap called

$ neofs-cli netmap --help
Operations with Network Map

Usage:
  neofs-cli netmap [flags]
  neofs-cli netmap [command]
...
and relevant help message
```
## Possible Solution
Implement expected behavior.

## Regression
No

## Your Environment
```
$ neofs-cli version
Version: v0.20.0 
Build: 2021-05-21T10:04:40+00:00 
Debug: false
```

### LINKED ISSUE #623: cli: Do not print help on any failure
Any error returned by implemented command executor leads to additional help output. It is useful when the user incorrectly formed a command, but not more.

### Solution
Replace providing the `RunE` calls of to `cobra.Command` (errors of which are actually lead to `help`) with `Run`. With this approach command misprints are still handled, but other errors are written user friendly.
For errors `cmd.PrintErr` function group, for logs - `cmd.Print*` w/o `Err`.

### PR DESCRIPTION
**Pull request by carpawell**
Closes #571 and closes #623.
Also, fixed error messages in object module("put" error message context not in `put` operations).

### CODE REVIEW

File: cmd/neofs-cli/modules/root.go [hunk 1]
```diff
@@ -107,7 +109,23 @@ func init() {
 
 	// Cobra also supports local flags, which will only run
 	// when this action is called directly.
-	// rootCmd.Flags().BoolP("toggle", "t", false, "Help message for toggle")
+	rootCmd.Flags().Bool("version", false, "Application version and NeoFS API compatibility")
+}
+
+func entryPoint(cmd *cobra.Command, _ []string) {
+	printVersion, err := cmd.Flags().GetBool("version")
+	if err == nil && printVersion {
```
Comment (cthulhu-rider) [inline cmd/neofs-cli/modules/root.go:117]: Looks like we can ignore error
```
printVersion, _ := cmd.Flags().GetBool("version")
if printVersion {}
```

**Comment by cthulhu-rider** [comment]
Everything is good. Small suggestion: replace
```
cmd.PrintErrln(fmt.Errorf(...))
```
with
```
cmd.PrintErrf(...)
```

**Review by cthulhu-rider — APPROVED** [review]

**Review by carpawell — COMMENTED** [review]
@cthulhu-rider, if use `cmd.PrintErrf(...)` then it is necessary to write smth like `cmd.PrintErrf("message: %s\n", err)` without `"%w"` and with `"\n"`, is it ok?

File: cmd/neofs-cli/modules/object.go [hunk 2]
```diff
@@ -407,34 +426,42 @@ func getObjectHeader(cmd *cobra.Command, _ []string) error {
...
 		}
 
-		return fmt.Errorf("can't put object: %w", err)
+		cmd.PrintErrln(fmt.Errorf("can't get object header: %w", err))
+		return
 	}
 
-	return saveAndPrintHeader(cmd, obj, cmd.Flag("file").Value.String())
+	if err = saveAndPrintHeader(cmd, obj, cmd.Flag("file").Value.String()); err != nil {
+		cmd.PrintErrln()
```
Comment (alexvanin) [inline cmd/neofs-cli/modules/object.go:437]:
```suggestion
		cmd.PrintErrln(err)
```

File: cmd/neofs-cli/modules/container.go [hunk 3]
```diff
@@ -454,49 +483,56 @@ var setExtendedACLCmd = &cobra.Command{
...
 		if err != nil {
-			return err
+			cmd.PrintErrln(err)
+			return
 		}
 
 		tok, err := getSessionToken(sessionTokenPath)
 		if err != nil {
-			return err
+			cmd.PrintErrln()
```
Comment (alexvanin) [inline cmd/neofs-cli/modules/container.go:515]:
```suggestion
			cmd.PrintErrln(err)
```

**Review by alexvanin — CHANGES_REQUESTED** [review]

File: cmd/neofs-cli/modules/object.go (hunk 2 above)
Comment (carpawell) [reply cmd/neofs-cli/modules/object.go]: fixed

**Review by alexvanin — APPROVED** [review]

**Pull request merged**